index.html

```diff
@@ -34,5 +34,93 @@ <h1 class="header-logo-wrapper">
         </div>
       </div>
     </section>
+    <section class="about">
+      <div class="about-container">
+        <h2 class="about-section-head">
+          What is &nbsp
```

Comment:
```suggestion
          What is&nbsp
```
2重にスペースが入ってしまうので、このようにすると解消できます。

Reply:
あとコードを見比べてもわからなかったのですが、自分がやった時には特に&nbspとか使わなくてもすんなり半角空いてたので、もっといい方法があるのかも。そこは二次レビューで言われるかもしれません。

Reply:
修正しました。

Reply:
なるほど・・・別の方法もあるかもなんですね。少し自分でも調べてみようと思います。

Reply:
自分は&nbspを知らなかったので，実装するために自ら調べられていて素晴らしいです！！🙌
&nbspを使用せずabout-section-headに効いているdisplay:flexを削除する方法もあります．